.github/workflows/build-lint-test.yml

```diff
@@ -37,34 +37,10 @@ jobs:
             exit 1
           fi
 
-  e2e:
```

Comment:
This one was breaking locally for the `getInterface` of Metamask SDK snap. Since this actually tests the SDK, and not this code base, I opted to leave it out. I can open an issue so we investigate later; wdyt? I'll re-enable and remove from the main CI until we wait for Metamask to fix it (there is no tracking issue there)

Reply:
Actually, ignore the previous comment, seems the problem is local; restored on CI